src/Components/Nav/Components/SignupModalEmail.js

```diff
@@ -1,145 +1,228 @@
-              <div>
-                <InfoInput placeholder="성(예: 홍)"></InfoInput>
-                {NavSvg.person}
-              </div>
-              <div>
-                <InfoInput placeholder="비밀번호 설정하기"></InfoInput>
-                {NavSvg.passwordInvisable}
-              </div>
-            </InfoSection>
-            <ValidationSection>
-              <div>
-                {NavSvg.validationNotPassed}
-                <span>비밀번호 보안 수준:</span>
-                <span>약함</span>
-              </div>
-              <div>
-                {NavSvg.validationNotPassed}
-                <span>
-                  비밀번호에 본인 이름이나 이메일 주소를 포함할 수 없습니다.
-                </span>
-              </div>
-              <div>
-                {NavSvg.validationNotPassed}
-                <span>최소 8자</span>
-              </div>
-              <div>
-                {NavSvg.validationNotPassed}
-                <span>숫자나 기호를 포함하세요</span>
-              </div>
-            </ValidationSection>
-            <SignupInfoBirth>
-              <h3>생일</h3>
-              <p>
-                만 18세 이상의 성인만 회원으로 가입할 수 있습니다. 생일은 다른
-                에어비앤비 이용자에게 공개되지 않습니다.
-              </p>
-              <BirthSelect width="41.6%">
-                <option>월</option>
-                <option value="1">1월</option>
-                <option value="2">2월</option>
-              </BirthSelect>
-              <BirthSelect width="25%">
-                <option>일</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-              </BirthSelect>
-              <BirthSelect width="33.3%">
-                <option>년</option>
-                <option value="2020">2020</option>
-                <option value="2019">2019</option>
-              </BirthSelect>
-            </SignupInfoBirth>
+export default function SignupModalEmail({ handleSignupEmail }) {
+  const [email, setEmail] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [password, setPassword] = useState("");
+  const [successSignup, setSuccessSignup] = useState(false);
+  const [year, setYear] = useState("");
+  const [month, setMonth] = useState("");
+  const [day, setDay] = useState("");
+
+  const submitSignup = (e) => {
+    if (!email) return alert("이메일이 필요합니다.");
+    else if (!firstName) return alert("이름이 필요합니다.");
+    else if (!lastName) return alert("성을 입력하세요.");
+    else if (!password) return alert("비밀번호를 입력하세요.");
+
+    if (email && firstName && lastName && password) {
+      fetch(`${signInAPI}/user/signup`, {
+        method: "post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          email: email,
+          first_name: firstName,
+          last_name: lastName,
+          password: password,
+          year: year,
+          month: month,
+          day: day,
+          platform: "airbnb",
```

Comment:
```suggestion
          email,
          first_name: firstName,
          last_name: lastName,
          password,
          year,
          month,
          day,
          platform: "airbnb",
```
굳이 줄여줄 필요 없긴 하지만 객체 단축 속성명을 이용하면 객체의 키값과 value에 담긴 변수명이 같을 경우 하나로 합쳐줄 수 있다는 사실을 언급해두고 싶네요!